### PR TITLE
audio and dynamic urls

### PR DESCRIPTION
**Pull request by jayridge**
Support for mixed media streams and dynamic urls.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,8 @@ RTSPrelay relays a video stream provided by an RTSP RECORD session to one or
 more PLAY sessions. It is intended as a proof of concept and has several
 limitations.
 
-* Only one RECORD session is supported.
-* Only h264 video is supported.
-
+* Ony h.264 video and aac audio streams are supported.
```
Comment (ploxiln) [inline README.md]: s/Ony/Only/

File: rtsprelay.c [hunk 2]
```diff
@@ -1,211 +1,311 @@
...
-    {NULL}
+	{"port", 'p', 0, G_OPTION_ARG_STRING, &port, "Listen port", "PORT"},
+	{NULL}
 };
 
-typedef struct
+typedef struct Relay {
+	GstRTSPServer *server;
+	/* using the word factory makes me feel ashamed */
+	GHashTable *factories;
```
Comment (ploxiln) [inline rtsprelay.c]: Gstreamer / GObject are OO so there's not much point in us OOP-shaming here ;)

Comment (jayridge) [reply rtsprelay.c]: whoops. usually remove any emotive comments before commit.

**Comment by ploxiln** [comment]
looks great 👍

File: Makefile [hunk 3]
```diff
@@ -1,11 +1,13 @@
 BLDDIR = build
 TARGET = rtsprelay
 
-CFLAGS = -g -O2 --std=c99 -Wall
+CFLAGS = -g -O0 --std=c99 -Wall
 LIBS =
 
-CFLAGS += `pkg-config --cflags gstreamer-1.0 gstreamer-plugins-base-1.0 gstreamer-rtsp-server-1.0`
-LDFLAGS += `pkg-config --libs gstreamer-1.0 gstreamer-plugins-base-1.0 gstreamer-rtsp-server-1.0`
+PKGS = libsoup-2.4 gstreamer-1.0 gstreamer-app-1.0 gstreamer-rtsp-1.0 gstreamer-rtsp-server-1.0
```
Comment (jayridge) [inline Makefile]: not using libsoup

**Pull request merged**